### ISSUE TITLE
Shift + Left/Right no longer jumps one second backwards/forwards along the timeline

### ISSUE THREAD

**Issue by clockworkpc** [OpenShot/openshot-qt#131]
In Openshot 1.4.3 I could use the keyboard shortcut SHIFT+Left to jump back one second on the timeline and SHIFT+Right to jump forward one second on the timeline.

If I'm not mistaken it was governed by this section in MainGTK.py:

```
    # check if frame-stepping or rewinding
    if single_frame == False:
        # SEEK BACKWARDS
        # calcualte new speed
        if current_speed >= 0:
            new_speed = -1
        else:
            new_speed = (current_speed * 2) 

        # set the new speed
        self.MyVideo.set_speed(new_speed)

        # update the preview tab label
        if new_speed == 1:
            self.lblVideoPreview.set_text(_("Video Preview"))
        else:
            self.lblVideoPreview.set_text(_("Video Preview (%sX)" % int(new_speed)))
    else:
        n_frames = 1
        if self._SHIFT:
            # step length is one second
            n_frames = int(round(self.project.fps()))
        # or just step 1 frame
        self.MyVideo.seek(position - n_frames)
```

**Comment by komackaj** [comment]
Multiple shortcuts not working. Related #138

**Comment by clockworkpc** [comment]
Still not fixed on Ubuntu 14.04:
I tried SHIFT + Right and SHIFT + Left, but no changed.

Loaded modules from installed directory: /usr/lib/python3/dist-packages/openshot_qt
      launch:INFO ------------------------------------------------
      launch:INFO    OpenShot (version 2.0.7)
      launch:INFO ------------------------------------------------
         app:INFO openshot-qt version: 2.0.7
         app:INFO libopenshot version: 0.1.1
         app:INFO platform: Linux-3.13.0-83-generic-x86_64-with-Ubuntu-14.04-trusty
         app:INFO processor: x86_64
         app:INFO machine: x86_64
         app:INFO python version: 3.4.3
         app:INFO qt5 version: 5.2.1
         app:INFO pyqt5 version: 5.2.1
    language:INFO Qt Detected Languages: ['en-AU', 'en']
    language:INFO LANG Environment Variable: en_AU.UTF-8
    language:INFO LOCALE Environment Variable: en_AU
    language:INFO Attempting to load qt_en_AU.UTF-8 in '/usr/share/qt5/translations'
    language:INFO Attempting to load qt_en in '/usr/share/qt5/translations'
    language:INFO Attempting to load en_AU.UTF-8/LC_MESSAGES/OpenShot in '/usr/lib/python3/dist-packages/openshot_qt/locale'
    language:INFO Attempting to load en/LC_MESSAGES/OpenShot in '/usr/lib/python3/dist-packages/openshot_qt/locale'
    language:INFO Attempting to load qt_en_AU in '/usr/share/qt5/translations'
    language:INFO Attempting to load qt_en in '/usr/share/qt5/translations'
    language:INFO Attempting to load en_AU/LC_MESSAGES/OpenShot in '/usr/lib/python3/dist-packages/openshot_qt/locale'
    language:INFO Successfully loaded en_AU/LC_MESSAGES/OpenShot in '/usr/lib/python3/dist-packages/openshot_qt/locale'
    language:INFO Exiting translation system (since we successfully loaded: en_AU)
project_data:INFO Setting default profile to HDV 720 24p
         app:INFO Setting font to /usr/lib/python3/dist-packages/openshot_qt/images/fonts/Ubuntu-R.ttf
         app:INFO Setting custom dark theme
    language:INFO Attempting to load qt_en_AU.UTF-8 in '/usr/share/qt5/translations'
    language:INFO Attempting to load qt_en in '/usr/share/qt5/translations'
    language:INFO Attempting to load en_AU.UTF-8/LC_MESSAGES/OpenShot in '/usr/lib/python3/dist-packages/openshot_qt/locale'
    language:INFO Attempting to load en/LC_MESSAGES/OpenShot in '/usr/lib/python3/dist-packages/openshot_qt/locale'
    language:INFO Attempting to load qt_en_AU in '/usr/share/qt5/translations'
    language:INFO Attempting to load qt_en in '/usr/share/qt5/translations'
    language:INFO Attempting to load en_AU/LC_MESSAGES/OpenShot in '/usr/lib/python3/dist-packages/openshot_qt/locale'
    language:INFO Successfully loaded en_AU/LC_MESSAGES/OpenShot in '/usr/lib/python3/dist-packages/openshot_qt/locale'
link SVGID_18_-5 hasn't been detected!
link SVGID_18_-5 hasn't been detected!
link SVGID_22_-5 hasn't been detected!
link SVGID_24_-2 hasn't been detected!
link SVGID_24_-2 hasn't been detected!
     metrics:INFO Track metric: {'cache-control': 'no-cache, no-store, must-revalidate', 'server': 'Golfe2', 'status': '200', 'content-location': 'http://www.google-analytics.com/collect?aip=1&an=OpenShot+Video+Editor&cd1=0.1.1&ua=Mozilla%2F5.0+%28X11%3B+Linux+x86_64%29+AppleWebKit%2F537.36+%28KHTML%2C+like+Gecko%29+Chrome%2F37.0.2062.120+Safari%2F537.36&cd=main-screen&cd4=5.2.1&v=1&cd2=3.4.3&cd3=5.2.1&aid=org.openshot.openshot-qt&ul=en-au&av=2.0.7&cd5=Ubuntu-14.04-trusty&t=screenview&tid=UA-4381101-5&cid=8d1ac492-05db-4dc3-8e5d-078b6558d9e7', 'pragma': 'no-cache', 'content-length': '35', 'last-modified': 'Sun, 17 May 1998 03:00:00 GMT', 'x-content-type-options': 'nosniff', 'age': '481740', 'expires': 'Mon, 01 Jan 1990 00:00:00 GMT', 'content-type': 'image/gif', 'date': 'Wed, 30 Mar 2016 12:01:12 GMT', 'access-control-allow-origin': '*'} (b'GIF89a\x01\x00\x01\x00\x80\xff\x00\xff\xff\xff\x00\x00\x00,\x00\x00\x00\x00\x01\x00\x01\x00\x00\x02\x02D\x01\x00;')
     metrics:INFO Track metric: {'cache-control': 'no-cache, no-store, must-revalidate', 'server': 'Golfe2', 'status': '200', 'content-location': 'http://www.google-analytics.com/collect?aip=1&an=OpenShot+Video+Editor&cd1=0.1.1&ua=Mozilla%2F5.0+%28X11%3B+Linux+x86_64%29+AppleWebKit%2F537.36+%28KHTML%2C+like+Gecko%29+Chrome%2F37.0.2062.120+Safari%2F537.36&cd=launch-app&cd4=5.2.1&v=1&cd2=3.4.3&cd3=5.2.1&aid=org.openshot.openshot-qt&sc=start&ul=en-au&av=2.0.7&cd5=Ubuntu-14.04-trusty&t=screenview&tid=UA-4381101-5&cid=8d1ac492-05db-4dc3-8e5d-078b6558d9e7', 'pragma': 'no-cache', 'content-length': '35', 'last-modified': 'Sun, 17 May 1998 03:00:00 GMT', 'x-content-type-options': 'nosniff', 'age': '481740', 'expires': 'Mon, 01 Jan 1990 00:00:00 GMT', 'content-type': 'image/gif', 'date': 'Wed, 30 Mar 2016 12:01:12 GMT', 'access-control-allow-origin': '*'} (b'GIF89a\x01\x00\x01\x00\x80\xff\x00\xff\xff\xff\x00\x00\x00,\x00\x00\x00\x00\x01\x00\x01\x00\x00\x02\x02D\x01\x00;')
     ui_util:INFO Initializing UI for MainWindow
     ui_util:INFO Binding event MainWindow:actionNew_trigger
     ui_util:INFO Binding event MainWindow:actionOpen_trigger
     ui_util:INFO Binding event MainWindow:actionSave_trigger
     ui_util:INFO Binding event MainWindow:actionUndo_trigger
     ui_util:INFO Binding event MainWindow:actionSaveAs_trigger
     ui_util:INFO Binding event MainWindow:actionImportFiles_trigger
     ui_util:INFO Binding event MainWindow:actionImportImageSequence_trigger
     ui_util:INFO Binding event MainWindow:actionRedo_trigger
     ui_util:INFO Binding event MainWindow:actionRemoveClip_trigger
     ui_util:INFO Binding event MainWindow:actionRemoveTransition_trigger
     ui_util:INFO Binding event MainWindow:actionExportVideo_trigger
     ui_util:INFO Binding event MainWindow:actionUploadVideo_trigger
     ui_util:INFO Binding event MainWindow:actionAddTrack_trigger
     ui_util:INFO Binding event MainWindow:actionPreferences_trigger
     ui_util:INFO Binding event MainWindow:actionPlay_trigger
     ui_util:INFO Binding event MainWindow:actionJumpStart_trigger
     ui_util:INFO Binding event MainWindow:actionRewind_trigger
     ui_util:INFO Binding event MainWindow:actionFastForward_trigger
     ui_util:INFO Binding event MainWindow:actionJumpEnd_trigger
     ui_util:INFO Binding event MainWindow:actionArrowTool_trigger
     ui_util:INFO Binding event MainWindow:actionSnappingTool_trigger
     ui_util:INFO Binding event MainWindow:actionAddMarker_trigger
     ui_util:INFO Binding event MainWindow:actionPreviousMarker_trigger
     ui_util:INFO Binding event MainWindow:actionNextMarker_trigger
     ui_util:INFO Binding event MainWindow:actionFilesShowAll_trigger
     ui_util:INFO Binding event MainWindow:actionFilesShowVideo_trigger
     ui_util:INFO Binding event MainWindow:actionFilesShowAudio_trigger
     ui_util:INFO Binding event MainWindow:actionFilesShowImage_trigger
     ui_util:INFO Binding event MainWindow:actionTransitionsShowAll_trigger
     ui_util:INFO Binding event MainWindow:actionTransitionsShowCommon_trigger
     ui_util:INFO Binding event MainWindow:actionEffectsShowAll_trigger
     ui_util:INFO Binding event MainWindow:actionEffectsShowVideo_trigger
     ui_util:INFO Binding event MainWindow:actionEffectsShowAudio_trigger
     ui_util:INFO Binding event MainWindow:actionTimelineZoomIn_trigger
     ui_util:INFO Binding event MainWindow:actionTimelineZoomOut_trigger
     ui_util:INFO Binding event MainWindow:actionTitle_trigger
     ui_util:INFO Binding event MainWindow:actionAnimatedTitle_trigger
     ui_util:INFO Binding event MainWindow:actionFullscreen_trigger
     ui_util:INFO Binding event MainWindow:actionAbout_trigger
     ui_util:INFO Binding event MainWindow:actionThumbnailView_trigger
     ui_util:WARNING Icon theme view-preview not found. Will use backup icon.
     ui_util:INFO Binding event MainWindow:actionDetailsView_trigger
     ui_util:WARNING Icon theme view-list-details not found. Will use backup icon.
     ui_util:INFO Binding event MainWindow:actionReportBug_trigger
     ui_util:INFO Binding event MainWindow:actionAskQuestion_trigger
     ui_util:INFO Binding event MainWindow:actionTranslate_trigger
     ui_util:INFO Binding event MainWindow:actionDonate_trigger
     ui_util:INFO Binding event MainWindow:actionHelpContents_trigger
     ui_util:INFO Binding event MainWindow:actionSimple_View_trigger
     ui_util:INFO Binding event MainWindow:actionAdvanced_View_trigger
     ui_util:WARNING Icon theme get-hot-new-stuff not found. Will use backup icon.
     ui_util:INFO Binding event MainWindow:actionFreeze_View_trigger
     ui_util:INFO Binding event MainWindow:actionUn_Freeze_View_trigger
     ui_util:INFO Binding event MainWindow:actionShow_All_trigger
     ui_util:INFO Binding event MainWindow:actionProfile_trigger
     ui_util:INFO Binding event MainWindow:actionAdd_to_Timeline_trigger
     ui_util:INFO Binding event MainWindow:actionPreview_File_trigger
     ui_util:INFO Binding event MainWindow:actionRemove_from_Project_trigger
     ui_util:INFO Binding event MainWindow:actionFile_Properties_trigger
     ui_util:INFO Binding event MainWindow:actionRemoveTrack_trigger
     ui_util:INFO Binding event MainWindow:actionRemoveMarker_trigger
     ui_util:INFO Binding event MainWindow:actionAddTrackAbove_trigger
     ui_util:INFO Binding event MainWindow:actionAddTrackBelow_trigger
     ui_util:INFO Binding event MainWindow:actionRemoveEffect_trigger
     ui_util:INFO Binding event MainWindow:actionSplitClip_trigger
     ui_util:INFO Binding event MainWindow:actionProperties_trigger
     ui_util:INFO Binding event MainWindow:actionRenameTrack_trigger
     ui_util:INFO Binding event MainWindow:actionUpdate_trigger
 files_model:INFO updating files model.
transition_model:INFO updating transitions model.
timeline_webview:INFO Qt Found!
timeline_webview:INFO $scope.Qt = true;
effects_model:INFO updating effects model.
effects_model:INFO category: Video
effects_model:INFO category: Video
effects_model:INFO category: Video
effects_model:INFO category: Video
effects_model:INFO category: Video
effects_model:INFO category: Video
effects_model:INFO category: Video
properties_model:INFO updating clip properties model.
preview_thread:INFO QThread Start Method Invoked
 main_window:INFO Clear all thumbnails: /home/alexandergarber/.openshot_qt/thumbnail
preview_thread:INFO initPlayer
 main_window:INFO Clear all animations: /home/alexandergarber/.openshot_qt/blender
 main_window:INFO Clear all backups: /home/alexandergarber/.openshot_qt/backup
project_data:INFO Setting default profile to HDV 720 24p
preview_thread:INFO refreshFrame
preview_thread:INFO self.player.Position(): 1
 main_window:INFO updateStatusChanged
         app:INFO Process command-line arguments: ['/usr/bin/openshot-qt']
 main_window:INFO updateStatusChanged
preview_thread:INFO onModeChanged
     version:INFO Found current version: {'transfer-encoding': 'chunked', 'server': 'Apache/2.2.22 (Ubuntu)', 'status': '200', 'content-location': 'http://www.openshot.org/version/json/', 'content-type': 'application/json', 'date': 'Tue, 05 Apr 2016 01:50:38 GMT'} (b'{"openshot_version": "2.0.7"}')
 main_window:INFO foundCurrentVersion: Found the latest version: 2.0.7
 main_window:INFO keyPressEvent: player.Position(): 1
files_listview:INFO Adding file: /home/alexandergarber/Videos/Projects/20160323_Flexibility/VIDEO_20160323_181926.mp4
project_data:INFO _set key: ['files'] values: {'top_field_first': True, 'type': 'FFmpegReader', 'audio_stream_index': 1, 'interlaced_frame': False, 'has_video': True, 'display_ratio': {'num': 4, 'den': 3}, 'height': 480, 'path': '/home/alexandergarber/Videos/Projects/20160323_Flexibility/VIDEO_20160323_181926.mp4', 'channels': 1, 'has_audio': True, 'acodec': 'aac', 'pixel_ratio': {'num': 65536, 'den': 65536}, 'media_type': 'video', 'audio_timebase': {'num': 1, 'den': 44100}, 'channel_layout': 4, 'vcodec': 'h264', 'video_bit_rate': 2098065, 'id': 'SI5XR0V7PV', 'has_single_image': False, 'fps': {'num': 132500, 'den': 8831}, 'duration': 21.36235809326172, 'file_size': '5602453', 'sample_rate': 44100, 'width': 640, 'video_timebase': {'num': 1, 'den': 90000}, 'video_stream_index': 0, 'audio_bit_rate': 96044, 'video_length': '318', 'pixel_format': 0} add: True partial: False remove: False
 files_model:INFO updating files model.
 main_window:INFO updateStatusChanged
project_data:INFO _set key: ['clips'] values: {'perspective_c3_x': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}, 'crop_y': {'Points': [{'co': {'Y': 0.0, 'X': 1.0}, 'interpolation': 2}]}, 'end': 21.36235809326172, 'perspective_c2_x': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}, 'crop_height': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}, 'crop_x': {'Points': [{'co': {'Y': 0.0, 'X': 1.0}, 'interpolation': 2}]}, 'anchor': 0, 'perspective_c1_x': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}, 'location_y': {'Points': [{'co': {'Y': 0.0, 'X': 1.0}, 'interpolation': 2}]}, 'location_x': {'Points': [{'co': {'Y': 0.0, 'X': 1.0}, 'interpolation': 2}]}, 'alpha': {'Points': [{'co': {'Y': 1.0, 'X': 1.0}, 'interpolation': 2}]}, 'file_id': 'SI5XR0V7PV', 'perspective_c1_y': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}, 'shear_y': {'Points': [{'co': {'Y': 0.0, 'X': 1.0}, 'interpolation': 2}]}, 'gravity': 4, 'time': {'Points': [{'co': {'Y': 0.0, 'X': 1.0}, 'interpolation': 2}]}, 'crop_width': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}, 'perspective_c2_y': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}, 'effects': [], 'title': 'VIDEO_20160323_181926.mp4', 'perspective_c3_y': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}, 'start': 0.0, 'id': 'OUEZJE90NH', 'volume': {'Points': [{'co': {'Y': 1.0, 'X': 1.0}, 'interpolation': 2}]}, 'scale': 1, 'waveform': False, 'duration': 21.36235809326172, 'perspective_c4_x': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}, 'layer': 4, 'scale_x': {'Points': [{'co': {'Y': 1.0, 'X': 1.0}, 'interpolation': 2}]}, 'shear_x': {'Points': [{'co': {'Y': 0.0, 'X': 1.0}, 'interpolation': 2}]}, 'image': '/home/alexandergarber/.openshot_qt/thumbnail/SI5XR0V7PV.png', 'wave_color': {'green': {'Points': [{'handle_type': 0, 'handle_left': {'Y': 123.0, 'X': 1.0}, 'handle_right': {'Y': 123.0, 'X': 1.0}, 'co': {'Y': 123.0, 'X': 1.0}, 'interpolation': 0}]}, 'blue': {'Points': [{'handle_type': 0, 'handle_left': {'Y': 255.0, 'X': 1.0}, 'handle_right': {'Y': 255.0, 'X': 1.0}, 'co': {'Y': 255.0, 'X': 1.0}, 'interpolation': 0}]}, 'red': {'Points': [{'handle_type': 0, 'handle_left': {'Y': 0.0, 'X': 1.0}, 'handle_right': {'Y': 0.0, 'X': 1.0}, 'co': {'Y': 0.0, 'X': 1.0}, 'interpolation': 0}]}, 'alpha': {'Points': [{'handle_type': 0, 'handle_left': {'Y': 255.0, 'X': 1.0}, 'handle_right': {'Y': 255.0, 'X': 1.0}, 'co': {'Y': 255.0, 'X': 1.0}, 'interpolation': 0}]}}, 'rotation': {'Points': [{'co': {'Y': 0.0, 'X': 1.0}, 'interpolation': 2}]}, 'reader': {'audio_stream_index': 1, 'type': 'FFmpegReader', 'interlaced_frame': False, 'has_video': True, 'display_ratio': {'num': 4, 'den': 3}, 'height': 480, 'path': '/home/alexandergarber/Videos/Projects/20160323_Flexibility/VIDEO_20160323_181926.mp4', 'channels': 1, 'has_audio': True, 'acodec': 'aac', 'pixel_ratio': {'num': 65536, 'den': 65536}, 'audio_timebase': {'num': 1, 'den': 44100}, 'channel_layout': 4, 'vcodec': 'h264', 'video_bit_rate': 2098065, 'top_field_first': True, 'has_single_image': False, 'fps': {'num': 132500, 'den': 8831}, 'duration': 21.36235809326172, 'file_size': '5602453', 'sample_rate': 44100, 'width': 640, 'video_timebase': {'num': 1, 'den': 90000}, 'video_stream_index': 0, 'audio_bit_rate': 96044, 'video_length': '318', 'pixel_format': 0}, 'position': 0.0, 'scale_y': {'Points': [{'co': {'Y': 1.0, 'X': 1.0}, 'interpolation': 2}]}, 'perspective_c4_y': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}} add: True partial: False remove: False
properties_model:INFO {'perspective_c3_x': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}, 'crop_y': {'Points': [{'co': {'Y': 0.0, 'X': 1.0}, 'interpolation': 2}]}, 'end': 21.36235809326172, 'perspective_c2_x': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}, 'crop_height': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}, 'crop_x': {'Points': [{'co': {'Y': 0.0, 'X': 1.0}, 'interpolation': 2}]}, 'anchor': 0, 'perspective_c1_x': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}, 'location_y': {'Points': [{'co': {'Y': 0.0, 'X': 1.0}, 'interpolation': 2}]}, 'location_x': {'Points': [{'co': {'Y': 0.0, 'X': 1.0}, 'interpolation': 2}]}, 'alpha': {'Points': [{'co': {'Y': 1.0, 'X': 1.0}, 'interpolation': 2}]}, 'file_id': 'SI5XR0V7PV', 'perspective_c1_y': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}, 'shear_y': {'Points': [{'co': {'Y': 0.0, 'X': 1.0}, 'interpolation': 2}]}, 'gravity': 4, 'time': {'Points': [{'co': {'Y': 0.0, 'X': 1.0}, 'interpolation': 2}]}, 'crop_width': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}, 'perspective_c2_y': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}, 'effects': [], 'title': 'VIDEO_20160323_181926.mp4', 'perspective_c3_y': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}, 'start': 0.0, 'id': 'OUEZJE90NH', 'volume': {'Points': [{'co': {'Y': 1.0, 'X': 1.0}, 'interpolation': 2}]}, 'scale': 1, 'waveform': False, 'duration': 21.36235809326172, 'perspective_c4_x': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}, 'layer': 4, 'scale_x': {'Points': [{'co': {'Y': 1.0, 'X': 1.0}, 'interpolation': 2}]}, 'shear_x': {'Points': [{'co': {'Y': 0.0, 'X': 1.0}, 'interpolation': 2}]}, 'image': '/home/alexandergarber/.openshot_qt/thumbnail/SI5XR0V7PV.png', 'wave_color': {'green': {'Points': [{'handle_type': 0, 'handle_left': {'Y': 123.0, 'X': 1.0}, 'handle_right': {'Y': 123.0, 'X': 1.0}, 'co': {'Y': 123.0, 'X': 1.0}, 'interpolation': 0}]}, 'blue': {'Points': [{'handle_type': 0, 'handle_left': {'Y': 255.0, 'X': 1.0}, 'handle_right': {'Y': 255.0, 'X': 1.0}, 'co': {'Y': 255.0, 'X': 1.0}, 'interpolation': 0}]}, 'red': {'Points': [{'handle_type': 0, 'handle_left': {'Y': 0.0, 'X': 1.0}, 'handle_right': {'Y': 0.0, 'X': 1.0}, 'co': {'Y': 0.0, 'X': 1.0}, 'interpolation': 0}]}, 'alpha': {'Points': [{'handle_type': 0, 'handle_left': {'Y': 255.0, 'X': 1.0}, 'handle_right': {'Y': 255.0, 'X': 1.0}, 'co': {'Y': 255.0, 'X': 1.0}, 'interpolation': 0}]}}, 'rotation': {'Points': [{'co': {'Y': 0.0, 'X': 1.0}, 'interpolation': 2}]}, 'reader': {'audio_stream_index': 1, 'type': 'FFmpegReader', 'interlaced_frame': False, 'has_video': True, 'display_ratio': {'num': 4, 'den': 3}, 'height': 480, 'path': '/home/alexandergarber/Videos/Projects/20160323_Flexibility/VIDEO_20160323_181926.mp4', 'channels': 1, 'has_audio': True, 'acodec': 'aac', 'pixel_ratio': {'num': 65536, 'den': 65536}, 'audio_timebase': {'num': 1, 'den': 44100}, 'channel_layout': 4, 'vcodec': 'h264', 'video_bit_rate': 2098065, 'top_field_first': True, 'has_single_image': False, 'fps': {'num': 132500, 'den': 8831}, 'duration': 21.36235809326172, 'file_size': '5602453', 'sample_rate': 44100, 'width': 640, 'video_timebase': {'num': 1, 'den': 90000}, 'video_stream_index': 0, 'audio_bit_rate': 96044, 'video_length': '318', 'pixel_format': 0}, 'position': 0.0, 'scale_y': {'Points': [{'co': {'Y': 1.0, 'X': 1.0}, 'interpolation': 2}]}, 'perspective_c4_y': {'Points': [{'co': {'Y': -1.0, 'X': 1.0}, 'interpolation': 2}]}}
properties_model:INFO updating clip properties model.
 main_window:INFO updateStatusChanged
preview_thread:INFO refreshFrame
preview_thread:INFO self.player.Position(): 1
timeline_webview:INFO Dropping ["SI5XR0V7PV"] in timeline.
project_data:INFO _set key: ['clips', {'id': 'OUEZJE90NH'}] values: {'start': 0, 'id': 'OUEZJE90NH', 'position': 1.6, 'layer': 4, 'image': '/home/alexandergarber/.openshot_qt/thumbnail/SI5XR0V7PV.png', 'end': 21.36235809326172} add: False partial: False remove: False
properties_model:INFO {'start': 0, 'id': 'OUEZJE90NH', 'position': 1.6, 'layer': 4, 'image': '/home/alexandergarber/.openshot_qt/thumbnail/SI5XR0V7PV.png', 'end': 21.36235809326172}
properties_model:INFO updating clip properties model.
 main_window:INFO updateStatusChanged
preview_thread:INFO refreshFrame
preview_thread:INFO self.player.Position(): 1
 main_window:INFO keyPressEvent: player.Position(): 1
 main_window:INFO keyPressEvent: player.Position(): 1
 main_window:INFO keyPressEvent: player.Position(): 2
 main_window:INFO keyPressEvent: player.Position(): 3
preview_thread:INFO previewFrame: 236
preview_thread:INFO self.player.Position(): 236
 main_window:INFO keyPressEvent: player.Position(): 236
 main_window:INFO keyPressEvent: player.Position(): 236
 main_window:INFO keyPressEvent: player.Position(): 237
 main_window:INFO keyPressEvent: player.Position(): 238
 main_window:INFO keyPressEvent: player.Position(): 239
 main_window:INFO keyPressEvent: player.Position(): 240
 main_window:INFO keyPressEvent: player.Position(): 241
 main_window:INFO keyPressEvent: player.Position(): 242

**Comment by DylanC** [comment]
@clockworkpc - Closing since this relates to a very old version of OpenShot. If this issue still exists in a newer version please comment on a newer ticket or open a new one. Thank you.
